src/mon/PaxosService.cc

```diff
@@ -382,20 +382,39 @@ void PaxosService::maybe_trim()
     return;
 
   version_t to_remove = trim_to - get_first_committed();
-  if (g_conf()->paxos_service_trim_min > 0 &&
-      to_remove < (version_t)g_conf()->paxos_service_trim_min) {
+  version_t paxos_service_trim_min = (version_t)g_conf()->paxos_service_trim_min;
```

Comment:
to access these conf values in the new, options.cc-only way, you can use g_conf()->get_val<uint64_t>("paxos_service_trim_min")

Reply:
Okay sure I'll change that.